### PR TITLE
Alpha into master

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Merge all alpha changes to master

### Breaking Changes

NA

### JIRA Link

NA

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

**Review by polymath-eric — APPROVED** [review]

**Comment by polymath-eric** [comment]
This should be a normal merge btw. No squash to preserve the history.

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "polymesh-subquery",
-  "version": "8.4.2",
+  "version": "8.5.06",
```
[the remark this answers is not in the document]
Comment (prashantasdeveloper) [reply package.json:3]: I would like to keep this as is for two reasons - 
* Lets us know that which change was the last change to be merged in master
* If we keep `8.5.0` the `8.5.04` and `8.5.06` migrations wont run. Also IMO it misguides status that version was reduced. Then next feat/fix change would bump master to 8.5.1 which seems the right way to do

[the remark this answers is not in the document]
Comment (polymath-eric) [reply package.json:3]: Makes sense if we get back to "normal" versioning going forward.

**Pull request merged**